extension/scripts/features/ranked-war-filter/ttRankedWarFilter.js

```diff
@@ -165,14 +165,26 @@
 		);
 	}
 
-	function filterRow(row, filters, individual) {
+	function filterRow(row, filters, individual, statsEstimatesOnly = false) {

```

Comment:
This boolean (and all the changes in this function) is not needed. You can just only pass the statsEstimates if needed (which is exactly what is being done).